### PR TITLE
Ignore platforms other than Linux for ffmpeg

### PR DESCRIPTION
**Pull request by prabhat00155**
Resolves https://github.com/pytorch/vision/issues/4402.
Part of https://github.com/pytorch/vision/issues/4392.

### CODE REVIEW

File: setup.py
Comment (datumbox) [inline setup.py:355]: For my info, do we need to disable also for Windows?

**Review by datumbox — APPROVED** [review]
LGTM @prabhat00155 in principle. I'm approving and I'll let you merge once you are happy with the CI and close the superseded PR at #4405.

File: setup.py
Comment (prabhat00155) [reply setup.py:355]: I tried it on Windows once and saw some failure. After fixing that, I'll re-enable it here.

**Pull request merged**
